### ISSUE TITLE
Don't copy entire /lib/udev directory (Reduce udev file usage)

### ISSUE THREAD

**Issue by fearedbliss** [fearedbliss/bliss-initramfs#18]
Recently I came across an issue on my laptop where my boot was being delayed by 1m52+ at the "Starting udev" stage (specifically 'udevadm settle'). My dmesg said the following:

bethany ~ # dmesg | grep udev
[ 3.371341] udevd[194]: starting version 238
[ 3.474168] udevd[205]: Process '/sbin/modprobe zfs' failed with exit code 2.
[ 3.484678] udevd[210]: Process '/lib/udev/VBoxCreateUSBNode.sh 189 1 e0' failed with exit code 1.
[ 3.486302] udevd[198]: Process '/lib/udev/VBoxCreateUSBNode.sh 189 2 ef' failed with exit code 1.
[ 63.704568] udevd[195]: seq 1156 '/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:00/PNP0C09:00/ACPI0003:00/power_supply/AC' is taking a long time
[ 63.704587] udevd[208]: spawned process 'lmt-udev auto' [209] is taking longer than 48 years 3 months ago to complete
[ 123.758309] udevd[208]: Process 'lmt-udev auto' failed with exit code 1.
[ 132.516351] udevd[391]: Process '/sbin/modprobe zfs' failed with exit code 2.
[ 133.396850] systemd[1]: Listening on udev Control Socket.

This problem isn't related to ZFS, Luks Encryption, nor the Linux Kernel since I tried 0.7.6, 0.7.8, with kernels 4.4.110, 4.9.75, and 4.14.33. Let alone I've used Gentoo on this laptop before and this didn't happen. Lastly, I installed Linux Mint on it to verify behavior, and Mint was fine.

Long story short, the problem was 'lmt-udev auto'. I had installed 'laptop-mode-tools' in Gentoo which contains some /lib/udev entries. These entries are currently being copied into the initramfs since bliss-initramfs copies the /lib/udev dir into itself.

I'll need to re-evaluate what the most critical files are for udev so that I can only copy only the core files necessary to boot up.

**Comment by fearedbliss** [comment]
Resolved with https://github.com/fearedbliss/bliss-initramfs/commit/8a22c8bdd3a99a7e2ce9a7ff4917fd03897baa4f, Will be included in 9.1.0.
